runtime-builder/build.sh

```diff
@@ -26,12 +26,12 @@ usage() { echo "Usage: $0 <project> <go_version>"; exit 1; }
 base_digest()
 {
   # We sometimes have issue with 'describe', use the --log-http flag to get more info if it fails.
```

Comment:
Is this comment still valid?  I think with the appropriate permissions, it should work now, right?